spec/features/visitor_sees_fruit_spec.rb

```diff
@@ -1,24 +1,12 @@
-require 'rails_helper'
```

Comment:
If this test is no longer used please delete the file.

Reply:
done